ui/app/services/flash-messages.ts

```diff
@@ -0,0 +1,10 @@
+import FlashMessages from 'ember-cli-flash/services/flash-messages';
+
+export default class PdsFlashMessages extends FlashMessages {
+  remove(id) {
+    let message = this.queue.findBy('id', id);
+    if (message) {
+      message._teardown();
```

Comment:
I don't think you'll need this yet (but maybe with a system type).

Reply:
Oh correct. I've updated it .